spec/item_transferer_spec.rb

```diff
@@ -3,25 +3,36 @@
 describe ItemTransferer do
   before do
     @request_body = {
-      'holdingTransfers' =>
+      'itemTransfers' =>
       [
-        {'source'       => {'owningInstitutionBibId' => 'aBibId', 'owningInstitutionHoldingsId' => 'aHoldingsId'},
-         'destination' =>  {'owningInstitutionBibId' => '.destinationBibId', 'owningInstitutionHoldingsId' => 'aHoldingsId'}}
+        {'source'       => {
+          'owningInstitutionBibId' => 'aBibId',
+          'owningInstitutionHoldingsId' => 'aHoldingsId',
+          'owningInstitutionItemId' => 'anItemId'
+        },
+         'destination' =>  {
+           'owningInstitutionBibId' => '.destinationBibId',
+           'owningInstitutionHoldingsId' => '.destinationBibId-ABC-123',
+           'owningInstitutionItemId' => 'anItemId'
+         }
+        }
       ],
       "institution" => "NYPL"
     }
 
     @item_transferer = ItemTransferer.new(
      api_key: 'fake_key',
      api_url: "http://example.com",
-     barcode_to_attributes_mapping: {'1234' => {'owningInstitutionBibId' => 'aBibId', 'owningInstitutionHoldingsId' => 'aHoldingsId'}},
+     barcode_to_attributes_mapping: {'1234' => {'owningInstitutionBibId' => 'aBibId', 'owningInstitutionHoldingsId' => 'aHoldingsId', 'owningInstitutionItemId' => 'anItemId'}},
      destination_bib_id: "destinationBibId"
    )
 
   end
 
   describe 'making calls' do
     it 'hits SCSB with the appropriate headers & body' do
+      expect(SecureRandom).to receive(:uuid).at_least(:once).and_return('ABC-123')
```

Comment:
Yup - the `expect()` is an expectation. If our implementation ever changed and `SecureRandom.uuid()` never got called - this would fail...the `and_return()` is a mocked response. 

This allows us to test our [expectations to HTTParty.post()](https://github.com/NYPL-discovery/scsb_item_updater/blob/01be12dd8357bc83e15de7adebacf3cab5d15ed9/spec/item_transferer_spec.rb#L36) that the POST body will contain "ABC-123"